Validation/RecoTau/plugins/DQMHistNormalizer.cc

```diff
@@ -18,15 +18,30 @@
 #include "FWCore/ServiceRegistry/interface/Service.h"
 
 //Regexp handling
-#include "classlib/utils/RegexpMatch.h"
-#include "classlib/utils/Regexp.h"
-
+#include <regex>
 #include <string>
 #include <vector>
 #include <map>
 
 using namespace std;
 
+// Three implementations were tested: char-by-char (this version),
+// using std::string::find + std::string::replace and std::regex_replace.
+// First one takes ~60 ns per iteration, second one ~85 ns,
+// and the regex implementation takes nearly 1 us
+std::string globToRegex(const std::string& s) {
+  std::string out;
+  out.reserve(s.size());
+  for (auto ch : s) {
+    if (ch == '*') {
+      out.push_back('.');
+      out.push_back('*');
+    }
+    out.push_back(ch);
```

Comment:
Doesn't this lead to `'*'` being inserted twice in the regex?

Reply:
yes , looks like `*` is  inserted twice

Reply:
ping @iarspider

Reply:
> a fix for this should go in 14.0.X too

ah it was only merged in 14.1.X, so no need to backport it for 14.0.X (my bad)